icu4c/source/i18n/unitsrouter.h

```diff
@@ -36,6 +37,35 @@ class U_I18N_API UnitsRouter {
 
 U_NAMESPACE_END
 
+namespace hugovdm_wip {
+// This namespace contains code from hugovdm that hasn't been reviewed by sffc
+// yet. It still needs thorough review and a "final resting place".
+
+using icu::CharString;
+using icu::MaybeStackVector;
+using icu::MeasureUnit;
+
+struct ConversionRateInfo {
+    CharString source;
+    CharString target;
+    CharString factor;
+    CharString offset;
+};
+
+struct UnitPreference {
+    UnitPreference() : geq(0) {}
```

Comment:
The default of `geq` should be `1`

Reply:
Strange: github is still telling me "younies requested changes" even though I marked this as "resolved".

Reply:
I suspect it's up to you to decide whether your requested changes have been suitably made, it's not up to me.